package.json

```diff
@@ -17,7 +17,7 @@
     "@types/node": "^20.4.2",
     "protobufjs": "^7.2.4",
     "protobufjs-cli": "^1.1.1",
-    "zeromq": "^6.0.0-beta.17"
+    "zeromq": "^6.0.0-beta.14"
```

Comment:
are we downgrading zeromq?

Reply:
Nope sorry, that was an accident. Fixed -> b67488d660be2482ff3dc7085b10a5378c233443